kubeflow/components/train.yaml

```diff
@@ -25,7 +27,7 @@ implementation:
     - |
       . /opt/genie-toolkit/lib.sh
 
-      parse_args "$0" "image s3_bucket owner dataset_owner task_name project experiment model load_from s3_datadir s3_model_dir" "$@"
+      parse_args "$0" "image s3_bucket owner dataset_owner task_name project experiment model load_from s3_datadir s3_model_dir skip_tensorboard" "$@"
```

Comment:
also add `train_iterations`?

Reply:
train_iterations is appended as additional argument to the bash script, because it gets passed directly to genienlp

Reply:
I see  ` --train_iterations, {inputValue: train_iterations} ` (line 94)  is passed into the bash script but it's not parsed here.

Reply:
Never mind. I see what you're saying now.